src/inc/internal/BundleWriterHelper.hpp

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+
+namespace MSIX {
+
+    struct PackageInfo
+        {
+            APPX_BUNDLE_PAYLOAD_PACKAGE_TYPE type;
+            UINT64 version;
+            std::string architecture;
+            std::string resourceId;
+            PCWSTR fileName;

```

Comment:
>PCWSTR [](start = 12, length = 6)

this should be a std::string. internally everything should be std::string and only when we marshal out a string the conversion to wstring should be done

Reply:
Updated all the fields to use standard std types.